app/ai/voice/agents/breeze_buddy/template/hooks.py

```
@@ -198,11 +201,30 @@ async def execute(
+                logger.info(
+                    f"No lead and no chat session for '{function_name}': "
+                    f"skipping outcome persistence."
+                )
+                return
+            try:
+                await update_chat_session_outcome(str(session_id), str(outcome))
+                logger.info(
+                    f"Persisted chat outcome '{outcome}' to session {session_id} "
+                    f"(function: '{function_name}')"
+                )
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)